### DISCUSSION TITLE
关于 Windows 资源管理器 的时间统计

### DISCUSSION THREAD

**Discussion by seyuk** [Planshit/Tai#34] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/57114382/150723392-50369560-2842-4a60-bdbc-4c1d91ca3aad.png)

我不知道为什么 Windows 资源管理器 会有这么长时间

**Comment by noberumotto** [comment, marked as answer]
这是一个bug，还在找原因中……可能要春节后才能处理了，抱歉
